packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -283,33 +284,35 @@ const App = (props) => {
 
                         <Box {...styles.headerWrapper}>
                             {!isCheckout ? (
-                                <Header
-                                    onMenuClick={onOpen}
-                                    onLogoClick={onLogoClick}
-                                    onMyCartClick={onCartClick}
-                                    onMyAccountClick={onAccountClick}
-                                    onWishlistClick={onWishlistClick}
-                                >
-                                    <HideOnDesktop>
-                                        <DrawerMenu
-                                            isOpen={isOpen}
-                                            onClose={onClose}
-                                            onLogoClick={onLogoClick}
-                                            root={categories?.[CAT_MENU_DEFAULT_ROOT_CATEGORY]}
-                                        />
-                                    </HideOnDesktop>
-
-                                    <HideOnMobile>
-                                        <ListMenu
-                                            root={categories?.[CAT_MENU_DEFAULT_ROOT_CATEGORY]}
-                                        />
-                                    </HideOnMobile>
-                                </Header>
+                                <>
+                                    <AboveHeader />
```

Comment:
If you want to have `<AboveHeader />` show for checkout, better move it up and out from the condition `!isCheckout ? ...`